1.0/docs/devguide/local-dom.md

```diff
@@ -234,14 +265,193 @@ Sometimes it's necessary to access the elements which have been distributed to a
     </template>
 
     // script
-    var div = Polymer.dom(xFoo).querySelector('div');
-    var content = Polymer.dom(xFoo.root).querySelector('content');
-    var distributed = Polymer.dom(content).getDistributedNodes()[0];
-    var insertedTo = Polymer.dom(div).getDestinationInsertionPoints()[0];
+    var lightDomDiv = Polymer.dom(xFoo).querySelector('div');
+    var insertionPoint = Polymer.dom(xFoo.root).querySelector('content');
+    var distributed = Polymer.dom(insertionPoint).getDistributedNodes()[0];
+    var insertedTo = Polymer.dom(lightDomDiv).getDestinationInsertionPoints()[0];
 
     // the following should be true:
-    assert.equal(distributed, div);
-    assert.equal(insertedTo, content)
+    console.log(distributed === lightDomDiv);
+    console.log(insertedTo === insertionPoint);
+
+Note that `getDistributedNodes` only returns useful results when called on a
+`<content>` element, and `getDestinationInsertionPoints` only returns useful
+results when called on an element that's being distributed.
+
+Because retrieving the children of a given content tag is a common pattern,
+the element API includes `getContentChildNodes` and `getContentChildren`
+utility methods. Both methods take a CSS selector that identifies a
+`<content>` element in the element's local DOM. (If the selector is omitted,
+both methods default to the first `<content>` element in the local DOM.)
+
+    <dom-module id="simple-content">
+      <template>
+        <content id="myContent"></content></span>
+      </template>
+      <script>
+        Polymer({
+          is: 'simple-content',
+          ready: function() {
+            var distributed = this.getContentChildren('#myContent');
+            console.log(distributed.length);
+          }
+        });
+      </script>
+    </dom-module>
+
+
+#### Effective children {#effective-children}
```

Comment:
Thanks. Yes, I'm trying to gradually rewrite stuff into a more task-oriented form (rather than a huge mass of conceptual info up front). We still need a lot of detailed conceptual stuff, but it can be moved later in the flow, so people only have to read it if they're curious and/or confused.